_help/README.md

```diff
@@ -0,0 +1,96 @@
+
+## The Help.html template
+tbd
+
+## The Help.md landing page
+tbd
+
+## The nav__sidenav.html component
+tbd
+
+## Adding a new help page
+1. Navigate to en.yml.
+    1. Find the `help` variable. Add the new page under an existing category:
+        ```
+          get-started:
+            overview: Overview
+            new-page-slug: New page title
+        ```
+        > Here, the new page is added to the get-started collection of pages.
+    1. Add the new page to the `help_pages` variable:
+        ```
+          get-started:
+            overview: |-
+              You can add or delete email addresses from your login.gov Your Account page...
+            new-page-slug: |-
+              This is the content of the new Help page...
+        ```
+        Remember to combine the yaml literal operator and the strip operator `|-` to keep line breaks but remove ending empty lines from the content.
+
+        **DO NOT USE HTML.** Content should include markdown syntax only. This applies to headings, lists, images and links. If special styling is required for content _within_ the help article, consider using general CSS selectors. Do not use ids created by markdown, because these ids will be translated and will not apply to translated pages.
+
+        To make sure that any links within the content, use the ``{{ site.baseurl/ }}`` prefix. This code converts to `login.gov/es` and `login.gov/fr` respectively.
+1. Repeat the above steps in es.yml. Use the same slug (untranslated) as the id.
+1. Repeat the above steps in fr.yml.
+
+## Adding a new help category
+1. Navigate to config.yml.
+1. Add the new category to the `help_pages` collection:
+    ```
+    help_pages:
+    - get-started
+    - trouble-signing-in
+    - new-category-slug
+    ```
+1. Navigate to en.yml.
+1. Add the new category to the `help_subpages` variable:
+    ```
+    help_subpages:
+      get-started: Get started with login.gov
+      trouble-signing-in: Trouble signing in?
+      new-category-slug: Title of new category
+    ```
+    These values are used by [...] 
+    
+    Titles in login.gov use sentence case instead of title case. Make sure category titles have the first letter capitalized.
+    
+    @TODO: Combine help_pages collection in config.yml with help_subpages. DRY.
+    @TODO: Rename help_subpages to help_catogories to improve semantic meaning.
+1. Add the new category to the `help_subpages` variable in es.yml and fr.yml.
+
+## Translations
+tbd
+
+## Redirects
+You'll notice there are several directories that aren't used on the brochure site:
+  * usajobs
+  * trusted-traveler-programs
+  * sam
+These pages were part of the former site, included in the navigation. Unfortunately, redirects on a Federalist site cannot be implemented on the server level. We have to create placeholders for these pages and specify where they should be redirected to.
```

Comment:
Needs an extra newline to avoid being collapsed into the last list item (see preview).

```suggestion

These pages were part of the former site, included in the navigation. Unfortunately, redirects on a Federalist site cannot be implemented on the server level. We have to create placeholders for these pages and specify where they should be redirected to.
```